MTV UI Overview and Settings dialog (#413)
kubev2v/forklift-documentation: documentation/modules/mtv-settings.adoc

// Module included in the following assemblies:
//
// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

:_content-type: PROCEDURE
[id="mtv-settings_{context}"]
= Configuring MTV settings

If you have Administrator privileges, you can access the *Overview* page and change the following settings in it:

[cols="1,1,1",options="header"]

> .{project-short} settings

Reviewer: this list aligns with the order the items appear in the UI which is good
but @yaacov, we should place the settings that are related to warm migration at the bottom - we consider dropping warm migration at some point...

Reviewer 2: added an issue:
https://github.com/kubev2v/forklift-console-plugin/issues/706

this issues, 705 and 706, are easy to solve, we can update the docs now, or wait until the fixes are merged ...

Reviewer: yeah, I'd wait for 706 but we can be explicit about the controller's main container before the fix for 705 gets in

Reviewer: I see the order was updated, resolving

|===
|Setting |Description |Default value

|Precopy internal (minutes)
|The interval at which a new snapshot is requested prior to initiating a warm migration
|60

|Snapshot polling interval (seconds)
|The frequency with which the system checks the status of snapshot creation or removal during warm migration
|10 seconds

|Max concurrent virtual machine migrations
|The maximum number of VMs that can be migrated simultaneously
|20

|Must gather cleanup after (hours)
|The duration for retaining 'must gather' reports before they are automatically deleted
|-1, which implies automatic cleanup is disabled.

|Controller CPU limit

> |The CPU limit allocated to the controller container

Reviewer 2: added an issue:
https://github.com/kubev2v/forklift-console-plugin/issues/705

// Module included in the following assemblies:
//
// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

:_content-type: PROCEDURE
[id="mtv-settings_{context}"]
= Configuring MTV settings

If you have Administrator privileges, you can access the *Overview* page and change the following settings in it:

[cols="1,1,1",options="header"]
.{project-short} settings
|===
|Setting |Description |Default value

|Max concurrent virtual machine migrations
|The maximum number of VMs per plan that can be migrated simultaneously
|20

|Must gather cleanup after (hours)
|The duration for retaining 'must gather' reports before they are automatically deleted
|Disabled.

|Controller main container CPU limit
|The CPU limit allocated to the main controller container
|500 m

|Controller main container Memory limit
|The memory limit allocated to the main controller container
|800 Mi

|Precopy internal (minutes)

> |The interval at which a new snapshot is requested prior to initiating a warm migration

Reviewer 3: ```suggestion
|The interval at which a new snapshot is requested before initiating a warm migration
```
Pedantic

|60

|Snapshot polling interval (seconds)
|The frequency with which the system checks the status of snapshot creation or removal during warm migration
|10

> 

Reviewer 3: Do you need this much which space... I do not mind but thought I should highlight

Author: Thanks. Extra lines result of shifting table rows' should have removed them when finished moving text. Interesting that bccutil did not object.
